src/app/routes/connector-ui/transfer-history-page/transfer-history-page/transfer-history-page.component.html

```diff
@@ -36,51 +36,48 @@
       class="transfer-history-table"
       mat-table
       [dataSource]="transferProcessesList.data.transferProcesses">
-      <ng-container matColumnDef="id">
-        <th *matHeaderCellDef mat-header-cell scope="col">ID</th>
-        <td *matCellDef="let item" mat-cell>{{ item.id }}</td>
+      <ng-container matColumnDef="direction">
+        <th *matHeaderCellDef mat-header-cell scope="col">Direction</th>
+        <td *matCellDef="let item" mat-cell>
+          <mat-icon mat-table-avatar>
+            {{ item.direction === 'PROVIDING' ? 'upload' : 'download' }}
+          </mat-icon>
+        </td>
       </ng-container>
-
-      <ng-container matColumnDef="creationDate">
-        <th *matHeaderCellDef mat-header-cell scope="col">Created</th>
+      <ng-container matColumnDef="assetId">
+        <th *matHeaderCellDef mat-header-cell scope="col">Asset</th>
         <td *matCellDef="let item" mat-cell>
-          {{ asDate(item.createdAt) }}
+          <div *ngIf="item.direction === 'PROVIDING'">
+            <a
+              class="link"
+              href="javascript:void(0)"
+              (click)="onAssetDetailsClick(item.transferProcessId)">
+              {{ item.assetName ?? item.assetId }}
```

Comment:
if it has a field "assetName", the backend should have already ensured, that it's non-null